pkg/porter/outputs.go

```diff
@@ -110,11 +112,9 @@ func NewDisplayOutputs(outputs claim.Outputs, format printer.Format) DisplayOutp
 		if err != nil {
 			continue
 		}
-		do.Type = outputType
 
-		// Try to figure out if this was originally a file output. Long term, we should find a way to find
-		// and crack open the invocation image to get the porter.yaml
-		if do.Type == "string" && schema.ContentEncoding == "base64" {
+		do.Type = outputType
```

Comment:
Can/should we use `outputType := extensions.GetParameterType(bun, schema)` above such that lines 117-119 below are no longer needed?